2024-03-27/minutes.md

```diff
@@ -1 +1,431 @@
+Does that work?
+
+I would also say regarding the timeline for this; there's no urgency.
+We also have to wait for Baseline to be on more pages before we can actually make a good educated decision, if we try to do that before then I think we'll be doing the work twice.
+
+[Estelle]
+There's about 6 different conversations about indicators and banners. So there's already all these discussions going on.
+
+[Ruth John]
+They're all captured in the Doc.
+I'd rather wait until we had a proposal and how we're going to move forward and then open the discussion and in all the related discussions, say 'Let's go and look at this discussion because this is our proposal'.
+
+[Estelle]
+I would love to at some point close all those other discussions and move on
+
+[Ruth John]
+Yeah, it's next on the list.
+
+### Writing Guidelines and Glossary
+
+**Discussion:** https://github.com/orgs/mdn/discussions/661
+
+[Brian Smith]
+The sections Glossary and Writing Guidelines living on islands. A bit more of a visibility to be recognized.
+
+[Ruth John]
+I think these are 2 separate things.
+I think where we link to the Writing Guidelines, where we link to the Glossary, are 2 separate discussions to be had because they're very separate things.
+In the discussion, Will very rightly points out that if we put Writing Guidelines at the top anywhere, our users are just going to think it's "how to write code" guidelines.
+So that's not something that we are going to do.
+
+Glossary; there are design and product considerations to find out how everyone feels about putting Glossary in that dropdown.
+It seems to be the consensus that people want it there.
+
+As far as the Writing Guidelines go.
+There's a lot of talk about where to put links and all the rest of it and the changes to be made.
+The key one for me was: when you click on contributing, you get taken to contributing.md on GitHub and then all the links in there move you back onto MDN.
+And then this kind of open kind of worms about where we put the contribution guidelines but I think they're too linked to the Writing Guidelines to really separate them.
+
+And we can't have the Writing Guidelines elsewhere because they contain so much styling from MDN itself that we'd lose all of that.
+If we read the Writing Guidelines as well.
+
+[Onkar]
+So I think it is better to move whatever there is in the GitHub `.md` files about contributing to the rendered pages and we host everything like other MDN docs.
+So, basically all the README files on GitHub (CONTRIBUTING.md), we move it into the content and host it.
+The guidelines.
+
+[Ruth John]
+Yeah, I think that we could stop the user flow from going to GitHub and back again.
+We need to keep repository specific stuff with the repository though.
+
+That was a conscious decision that we made when we were doing all the contribution docs because otherwise we're gonna have 2 places to like list our 50 repositories and put all the guidelines for all of them on MDN.
```

Comment:
```suggestion
That was a conscious decision that we made when we were doing all the contribution docs because otherwise we're gonna have 2 places to list our 50 repositories and put all the guidelines for all of them on MDN.
```